pkg/wrappers/mysqlcluster/status.go

```diff
@@ -0,0 +1,161 @@
+/*
+Copyright 2018 Pressinfra SRL
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package mysqlcluster
+
+import (
+	"time"
+
+	"github.com/golang/glog"
+
+	core "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	api "github.com/presslabs/mysql-operator/pkg/apis/mysql/v1alpha1"
+)
+
+// UpdateStatusCondition sets the condition to a status.
+// for example Ready condition to True, or False
+func (c *MysqlCluster) UpdateStatusCondition(condType api.ClusterConditionType,
```

Comment:
Should be named `UpdateConditionStatus` because it updates a condition's status and the signature should be on a single line if it fits  in 170 chars.